debugger/breakpoints.ml

```diff
@@ -170,9 +170,12 @@ let rec new_breakpoint =
            incr breakpoint_number;
            insert_position event.ev_pos;
            breakpoints := (!breakpoint_number, event) :: !breakpoints);
-      printf "Breakpoint %d at %d: %s" !breakpoint_number event.ev_pos
-             (Pos.get_desc event);
-      print_newline ()
+      if !Parameters.breakpoint
+      then begin
```

Comment:
To be consistent with the codebase, you should put the `then begin` at the end of the previous line.